docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -365,6 +365,95 @@ Continue configuring your Kubernetes provider by clicking **Add region** and com
+  For more information, see [Toleration API](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#toleration-v1-core).
+
+    ```yml
+    ## Consider node has following taint.
+    ## kubectl taint nodes node1 dedicated=experimental:NoSchedule-
+
+    master:
+      tolerations:
+      - key: dedicated
+        operator: Equal
+        value: experimental
+        effect: NoSchedule
+
+    tserver:
+      tolerations: []
+    ```
+
+  - You can use `nodeSelector` to schedule Master and TServer pods on dedicated nodes. For more information, see [Node Selector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector).
+
+    ```yml
+    ## The pod will get scheduled on a node that has a topology.kubernetes.io/zone=asia-south2-a label.
+    nodeSelector:
+      topology.kubernetes.io/zone: asia-south2-a
+    ```
+
+  - Add `affinity` in Master and TServer pods. The `affinity` allows the Kubernetes scheduler to place a pod on a set of nodes or a pod relative to the placement of other pods. You can use `nodeAffinity` rules to control pod placements on a set of nodes. In contrast, `podAffinity` or `podAntiAffinity` rules provide the ability to control pod placements relative to other pods. For more information, see [Affinity API](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#affinity-v1-core).
+
+    ```yml
+    ## Following example can be used to prevent scheduling of multiple master pods on single kubernetes node.
+    master:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - "yb-master"
+            topologyKey: kubernetes.io/hostname
+
+    tserver:
+      affinity: {}
+    ```
+
+  - Add `annotations` to Master and Tserver pods. The Kubernetes `annotations` can attach arbitrary metadata to objects. For more information, see [Annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/).
+
+    ```yml
+    master:
+      podAnnotations:
+        application: "yugabytedb"
+
+    tserver:
+      podAnnotations: {}
+    ```
+
+  - Add `labels` to Master and Tserver pods. The Kubernetes `labels` are key/value pairs attached to objects. The `labels` are used to specify identifying attributes of objects that are meaningful and relevant to you. For more information, see [Labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/).
+
+    ```yml
+    master:
+      podLabels:
+        environment: production
+        app: yugabytedb
```

Comment:
```suggestion
        app: yugabytedb
        prometheus.io/scrape: true
```

Just adding a popular label, if anyone is trying to do something similar, they will find it quickly.